src/lib/loaders/api/index.js

```diff
@@ -4,6 +4,7 @@ import config from "config"
 import convection from "lib/apis/convection"
 import delta from "lib/apis/delta"
 import diffusion from "lib/apis/diffusion"
+import galaxy from "lib/apis/galaxy"
```

Comment:
I was surprised to find we have almost ~12k `PartnerShow` records in production pointing to a Galaxy partner! (Used for ref shows)

Reply:
📈 👨‍💼

Reply:
Any idea how much memory was being retained here? Was it ~12k IDs?

Reply:
No, this is  just an endpoint per id. I'm just surprised that we had that many different entries.